app/helpers/multi_languages_helper.rb

```diff
@@ -172,15 +173,15 @@ def display_in_multiple_languages(label)
                                                 closable: true)
     end
 
-
-
     label = label.to_h.reject { |key, _| %i[links context].include?(key) } if label.is_a?(OpenStruct)
 
     if label.is_a?(String)
       content_tag(:p, label)
     elsif label.is_a?(Array)
-      content_tag(:div) do
-        raw(label.map { |x| content_tag(:div, x) }.join)
+      list_items_component(max_items: show_max) do |r|
```

Comment:
there is a bug here,  `show_max` and `style_as_badge` variables are not present.

Reply:
fixed here https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/930/files#diff-6aa19eac09e6ab23032652dafa4797b1fc40860e9bb1e4ffcb0b4948b116539e

Reply:
fixed